client/signup/step-wrapper/style.scss

```diff
@@ -136,6 +136,7 @@
 		.step-wrapper__header {
 			display: block;
 			margin-top: 0;
+			padding-right: 20px;
```

Comment:
I think we don't need this on mobile?

Reply:
Oops I forgot about that! Thanks for the reminder